src/main/java/com/epam/aidial/core/controller/ControllerSelector.java

```diff
@@ -211,8 +212,8 @@ private static Controller selectPost(Proxy proxy, ProxyContext context, String p
         return null;
     }
 
-    private static Controller selectDelete(Proxy proxy, ProxyContext context, String path) {
-        Matcher match = match(PATTERN_FILES, path);
+    private static Controller selectDelete(Proxy proxy, ProxyContext context, String providedPath, String decodedPath) {
+        Matcher match = match(PATTERN_FILES, providedPath);
```

Comment:
Please write some tests. You use Base64 encoding for buckets. It contains "/". How do you distinguish a slash in a bucket's name and in a path?

Reply:
I would use Base58.

Reply:
changed